src/Epsagon.Dotnet.Instrumentation/EpsagonLabels.cs

```diff
@@ -5,21 +5,54 @@
 
 namespace Epsagon.Dotnet.Instrumentation
 {
-    public class EpsagonLabels
-    {
-        public static void Add(string key, string value)
-        {
-            if (!(GlobalTracer.Instance?.ActiveSpan is Span span)) return;
-            var tags = span.GetTags();
-
-            Dictionary<string, string> labelsDict = new Dictionary<string, string>();
-            if (tags.TryGetValue("epsagon.labels", out var labels))
-            {
-                labelsDict = JsonConvert.DeserializeObject<Dictionary<string, string>>(labels.ToString());
-            }
-
-            labelsDict[key] = value;
-            span.SetTag("epsagon.labels", JsonConvert.SerializeObject(labelsDict));
-        }
-    }
+	public class EpsagonLabels
+	{
+		public static void Add(string key, string value)
+		{
+			AddLabel(
```

Comment:
Can you fix formatting in function calls as well?

Reply:
Yep, done.